### PR TITLE
chore: writestats consolidate counting

### PR DESCRIPTION
**Pull request by npetzall**

### CODE REVIEW

File: src/main/java/org/schemaspy/view/WriteStats.java [hunk 1]
```diff
@@ -27,22 +27,17 @@
  * @author John Currier
  */
 public class WriteStats {
```
Comment (jesperolsson-se) [inline src/main/java/org/schemaspy/view/WriteStats.java:29]: We might as well rename the class now, which allows us to (re)use it more generally. It's a `Tally`, maybe?

File: src/main/java/org/schemaspy/view/WriteStats.java [hunk 2]
```diff
@@ -27,22 +27,17 @@
...
-            ++numViews;
-        else
-            ++numTables;
+    public void wrote() {
+        written++;
     }
 
-    public int getNumTablesWritten() {
-        return numTables;
+    public void wrote(int number) {
```
Comment (jesperolsson-se) [inline src/main/java/org/schemaspy/view/WriteStats.java:36]: This method is introduced in this PR, but nothing is using it?

File: src/main/java/org/schemaspy/view/WriteStats.java (hunk 1 above)
Comment (jesperolsson-se) [inline src/main/java/org/schemaspy/view/WriteStats.java:29]: Add unit tests.

Comment (npetzall) [reply src/main/java/org/schemaspy/view/WriteStats.java]: Used LongAdder jdk class

**Comment by npetzall** [comment]
Not out of the woods, but it's replaced with LongAdder and it's not used in Summary which is sort of nice.

**Pull request merged**